bindings/java/native/src/classes/account_manager/acc_manager.rs

```diff
@@ -43,6 +43,7 @@ pub fn signer_type_enum_to_type(signer_type: AccountSignerType) -> SignerType {
         AccountSignerType::LedgerNanoSimulator => SignerType::LedgerNanoSimulator,
 
         // Default will only happen when we compile without any features...
+        #[cfg(not(all(feature = "stronghold", feature = "ledger-nano", feature = "ledger-nano-simulator",)))]
```

Comment:
Was there a clippy warning or what's the reason to add this?

Reply:
Only when `--all-features` is turned on will this raise a warning, as the above branches have already matched all possible options of `AssountSignerType`, and this catch-all is useless.